### ISSUE TITLE
Non-necrotizing granuloma

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6403]
**Preferred term label:**
Non-necrotizing granuloma

**Synonyms**
Non-necrotizing granulomatous inflammation

**Definition (free text, please give PubMed ID)**
A granuloma which is not associated with necrotic changes.
PMID: 32476832

**Parent term (use hpo.jax.org/app)**
Granuloma

**Comment by pnrobinson** [comment]
added as HP_0033805
